Makefile

```diff
@@ -25,6 +25,8 @@ clean-pyc:
 lint: build
 	docker-compose run app flake8 --max-line-length 100 .
 	docker-compose run app yamllint repositories.yaml .circleci
+	docker-compose run app python -m black .
```

Comment:
Linting should check for potential errors, instead of actually formatting. There's a `black` command-line flag that will do this, but the behavior here will be to format the files instead.